string event
string time
string cell_id
string comment
